airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/task/internal/HeartbeatTask.kt

```diff
@@ -20,10 +21,13 @@ class HeartbeatTask(
 ) : Task {
     override val terminalCondition: TerminalCondition = OnEndOfSync
 
+    private val log = KotlinLogging.logger {  }
+
     override suspend fun execute() {
         while (true) {
             delay(config.heartbeatIntervalSeconds * 1000L)
             try {
+                log.info { "Sending heartbeat" }
```

Comment:
I also added some logging in my PR. I think I'll back that out if you intend to merge this.